<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::EditUndo"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="captchaLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="usernameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="captchaLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Please enter your username and password.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshCaptcha">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentRevert"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
